core/src/main/scala/com/avast/clients/storage/storage.scala

```diff
@@ -0,0 +1,67 @@
+package com.avast.clients
+
+import better.files.File
+import cats.data.EitherT
+import cats.{~>, Monad}
+import com.avast.scala.hashes.Sha256
+import mainecoon.FunctorK
+
+import scala.language.higherKinds
+
+package object storage {
```

Comment:
Why is this class needed? Where is it used pls?

Reply:
`import c.a.clients.storage._`